File: backend/src/main/java/com/evalorithm/dto/response/AuthResponse.java
package com.evalorithm.dto.response;

import com.evalorithm.enums.Role;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class AuthResponse {

    private String accessToken;
    private String refreshToken;
    @Builder.Default
    private String tokenType = "Bearer";
    private long expiresIn;
    private Long userId;
    private String email;
    private String firstName;
    private String lastName;
    private Role role;
}

File: android/app/src/main/java/com/evalorithm/data/model/AuthResponse.kt
package com.evalorithm.data.model

data class AuthResponse(
    val accessToken: String,
    val refreshToken: String,
    val tokenType: String,
    val expiresIn: Long,
    val userId: Long,
    val email: String,
    val firstName: String,
    val lastName: String,
    val role: String
)
